### PR TITLE
Enable linking of email and Facebook accounts

### PR DESCRIPTION
**Pull request by bt-s**
I am really starting to love React hooks and Firebase!

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 const Button = props => (
```
Comment (Envl) [inline src/components/Button/Button.js:4]: What's the difference between using built-in <button> and our Button?

File: src/components/Account/DefaultLoginToggle.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    </form>
+  );
+};
+
+DefaultLoginToggle.propTypes = {
+  onlyOneLeft: PropTypes.bool,
+  isEnabled: PropTypes.bool,
+  signInMethod: PropTypes.object,
+  onUnlink: PropTypes.func
+};
```
Comment (Envl) [inline src/components/Account/DefaultLoginToggle.js:57]: Should there be a line like this? 
`onLink: PropTypes.func`

**Review by Envl — APPROVED** [review]
I've not grasped all the details but the general idea of this whole project. 
Everything works, that's nice.
Hopefully, I can catch up in these two or three days.

I'll start to adapt all our code using @decorators to define PropTypes at the beginning of a class to make it easier to read.

File: src/components/Button/Button.js (hunk 1 above)
Comment (bt-s) [reply src/components/Button/Button.js:4]: Our button has already some default classes set to it and makes it easier to use icons.

File: src/components/Account/DefaultLoginToggle.js (hunk 2 above)
Comment (bt-s) [reply src/components/Account/DefaultLoginToggle.js:57]: Well-spotted! I will add it

**Pull request merged**